optimizely/notification/manager.go

```diff
@@ -51,9 +51,7 @@ func (am *AtomicManager) Add(newHandler func(interface{})) (int, error) {
 // Remove removes handler with the given id
 func (am *AtomicManager) Remove(id int) {
 	handlerID := uint32(id)
-	if _, ok := am.handlers[handlerID]; ok {
```

Comment:
can we keep that and log a message if the id is not in the map ? just return within the "if" statement, and then log after it.

Reply:
delete works even if key doesn't exist because of which linter was throwing error while using the above condition. We can check for !ok, log and return, and then call delete on the dictionary?

Reply:
hmm, I know, but my point was to send the warning in case the id did not exist and you try to delete something that did not exist - to me that scenario should not happen. I guess that's fine then if linters complain. No worries.

Reply:
> hmm, I know, but my point was to send the warning in case the id did not exist and you try to delete something that did not exist - to me that scenario should not happen. I guess that's fine then if linters complain. No worries.

yes i got your point, just bound by the linter warning, thanks.

Reply:
can you try this: 
```
if _, ok := am.handlers[handlerID]; ok {
		delete(am.handlers, handlerID)
                 return
	}
        // log here 
```

Reply:
`return` statement seems to have done the trick for avoiding linter warning, thanks.

Reply:
great, smart  linters :), they figured out that nothing was done when item not in a map.